app/components/forms/order-form.js

```diff
@@ -0,0 +1,110 @@
+import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
+import { countries } from 'open-event-frontend/utils/dictionary/demography';
+
+const { Component, computed } = Ember;
+
+export default Component.extend(FormMixin, {
+  getValidationRules() {
+    var validationRules = {};
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+      fields : {
+        firstname: {
+          for(var i = 1; i <= 3; i++){
```

Comment:
@harshitagupta30 you _cannot_ put a loop inside an object. I really suggest you read up on the basics of javascript. Because this is something any basic javascript programmer should know about. Please do read up. 

This should be done by something like : https://pastebin.com/MweQB46x